compiler/rustc_metadata/src/rmeta/decoder/cstore_impl.rs

```diff
@@ -56,7 +56,7 @@ macro_rules! provide {
                 $compute
             })*
 
-            *providers = Providers {
+            *providers = ExternProviders {
```

Comment:
Is this provide exhaustive now? Should it be made so?

Reply:
Due to macro implementation issues I am using `()` as type for queries that don't have a separate extern provider.